en/docs/apis/create-and-publish-your-first-api.md

```diff
@@ -3,43 +3,41 @@
 Choreo’s API management capabilities allow you to create, publish, and manage all aspects of an API and its lifecycle.
 This quick start guide walks you through the steps to quickly create and publish an API using Choreo. 
```

Comment:
Should we explicitly mention that we are creating a REST API?

Reply:
+1